Rakefile

```diff
@@ -1,8 +1,13 @@
 #!/usr/bin/env rake
 require 'bundler/gem_tasks'
 require 'rspec/core/rake_task'
+require 'rubocop/rake_task'
 
 RSpec::Core::RakeTask.new
+RuboCop::RakeTask.new do |task|
```

Comment:
You don't need the block here at all. Rubocop will pick up the `.rubocop.yml` file by default and will include all lintable files. You're actually missing some files with your patterns definiton.